From d80b87cf9cf7987c6daf8a61f446e219131110e1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Felix=20Paul=20K=C3=BChne?= <felix@feepk.net>
Date: Fri, 29 Nov 2019 11:26:35 +0100
Subject: [PATCH 24/24] apple/build: rename installation folder for contrib

This strips duplicate information and makes the folder a lot easier to parse.
---
 extras/package/apple/build.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/extras/package/apple/build.sh b/extras/package/apple/build.sh
index f2ceb2a971..4fc9ea77fd 100755
--- a/extras/package/apple/build.sh
+++ b/extras/package/apple/build.sh
@@ -429,7 +429,7 @@ validate_architecture "$VLC_HOST_ARCH"
 # Set pseudo-triplet
 readonly VLC_PSEUDO_TRIPLET="${VLC_HOST_ARCH}-apple-${VLC_HOST_PLATFORM}_${VLC_DEPLOYMENT_TARGET}"
 # Contrib install dir
-readonly VLC_CONTRIB_INSTALL_DIR="$VLC_BUILD_DIR/contrib/$VLC_PSEUDO_TRIPLET"
+readonly VLC_CONTRIB_INSTALL_DIR="$VLC_BUILD_DIR/contrib/${VLC_HOST_ARCH}-${VLC_APPLE_SDK_NAME}"
 # VLC install dir
 readonly VLC_INSTALL_DIR="$VLC_BUILD_DIR/vlc-${VLC_APPLE_SDK_NAME}-${VLC_HOST_ARCH}"
 
-- 
2.21.0 (Apple Git-122.2)

